miio/device.py

```diff
@@ -119,7 +119,7 @@ def raw_command(self, command, parameters):
     @command(
         skip_autodetect=True,
     )
-    def info(self, *, skip_cache=False) -> DeviceInfo:
+    def info(self, skip_cache=False) -> DeviceInfo:
```

Comment:
```suggestion
    def info(self, *, skip_cache=False) -> DeviceInfo:
```
The kwarg-only parameter definition is intentional, as it forces one to type `info(skip_cache=True)` to make it unambiguous what is really wanted.

Reply:
TIL; thanks!

Done in separate PR